Cargo.toml

```diff
@@ -27,6 +27,7 @@ protobuf = "2"
 clap = "2.31"
 log = "0.4"
 log4rs = "0.8"
+rust-crypto = "0.2"
```

Comment:
If this is only used for testing it can be moved to a [dev-dependencies] section of the Cargo.toml file.